lib/github_backend.rb

```diff
@@ -16,6 +16,15 @@ def initialize(_args = {})
     @logger.level = Logger::DEBUG unless ENV['RACK_ENV'] == 'production'
   end
 
+  def testable_repos
+    projects = JSON.parse(Faraday.get('https://team-api.18f.gov/public/api/projects/').body)
+
+    projects.keys.inject([]) do |result, key|
```

Comment:
`projects` is a Hash. `items` is not a method you can call on a Hash. The reason why I have to use `inject` is because the API is returning a Hash instead of an Array of results. See https://github.com/18F/team-api/issues/87.

If it were an Array, I could do `projects.select { |p| p['testable'] }.map { |p| p['github'].first }`